build.sh

```diff
@@ -1,4 +1,10 @@
 MONO_SDK=$(pwd)/mono-wasm-sdk
 
-dotnet build Inzynierka.sln
+platform="$(uname -s)"
+case "${platform}" in
+    MSYS*)     buildCmd="dotnet build";;
+    *)          buildCmd="xbuild";;
```

Comment:
I got a message that `xbuild` is deprecated while using it and that I should use `msbuild` :smile: Could you verify if you get the same message and if so, change the script?

Reply:
I have it after updating mono 👍 .